### PR TITLE
v1.57.0 Release Notes

### PR DESCRIPTION
**Pull request by jshaughn**
https://deploy-preview-583--kiali.netlify.app/news/release-notes/

### CODE REVIEW

File: content/en/news/release-notes.md [hunk 1]
```diff
@@ -6,6 +6,44 @@ weight: 1
...
+
+* [Revisit "round timeseries on client-side with significant decimals"](https://github.com/kiali/kiali/issues/5432)
+* [(CI) Test flake - Service details page](https://github.com/kiali/kiali/issues/5358)
+* [(hack) setup kind in ci finalising with error](https://github.com/kiali/kiali/issues/5441)
+* [(CI) Test flake - workload details](https://github.com/kiali/kiali/issues/5407)
+* [(CI) Test Flake - TestConcurrentClientExpiration](https://github.com/kiali/kiali/issues/5196)
+* [(CI) Test flake - The degraded status of a service is reported in the list of service](https://github.com/kiali/kiali/issues/5226)
+* [(CI) Test flake - Sidebar toggle](https://github.com/kiali/kiali/issues/5437)
+* [Upgrade operator base image to 4.11](https://github.com/kiali/kiali/issues/5459)
+* [Change istioctl install hack script to default to single cluster settings](https://github.com/kiali/kiali/issues/5248)
```
Comment (jmazzitelli) [inline content/en/news/release-notes.md:35]: do we care to call this out (as well as line 29 - that's the hack script for our CI testing). I don't think people care.

File: content/en/news/release-notes.md [hunk 2]
```diff
@@ -6,6 +6,44 @@ weight: 1
...
+
+Upgrade Notes:
+
+The [improved control plane card on the Overview page](https://github.com/kiali/kiali/issues/5167) make use of previously unused
```
Comment (jmazzitelli) [inline content/en/news/release-notes.md]:
```suggestion
The [improved control plane card on the Overview page](https://github.com/kiali/kiali/issues/5167) makes use of previously unused
```

File: content/en/news/release-notes.md [hunk 3]
```diff
@@ -6,6 +6,44 @@ weight: 1
...
+metrics.  If these metrics have been removed from your environment you will need to add them back for the feature to work.  As a result,
```
Comment (jmazzitelli) [inline content/en/news/release-notes.md:40]: I think you should be specific here. Tell them which metrics you are referring to (don't make them look in the list of metrics in our documentation and try to figure out which one they removed but needs to put back in).

**Review by jmazzitelli — CHANGES_REQUESTED** [review]

File: content/en/news/release-notes.md (hunk 3 above)
Comment (jshaughn) [reply content/en/news/release-notes.md:40]: Fine.

File: content/en/news/release-notes.md (hunk 1 above)
Comment (jshaughn) [reply content/en/news/release-notes.md:35]: Its fine as is.

**Review by jmazzitelli — APPROVED** [review]

**Pull request merged**